patches/src/progname.c.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/src/progname.c b/src/progname.c
+index 83e8652..07c0528 100644
+--- a/src/progname.c
++++ b/src/progname.c
+@@ -65,7 +65,7 @@ getprogname(void)
+ 	/* getexecname(3) returns an absolute pathname, normalize it. */
+ 	if (__progname == NULL)
+ 		setprogname(getexecname());
+-#elif defined(_AIX)
++#elif defined(_AIX) 
+ 	if (__progname == NULL) {
+ 		struct procentry64 procs;
+ 		pid_t pid = getpid ();
+@@ -73,6 +73,9 @@ getprogname(void)
+ 		if (getprocs64(&procs, sizeof procs, NULL, 0, &pid, 1) > 0)
+ 			__progname = strdup(procs.pi_comm);
+ 	}
++#elif defined(__MVS__)
++	/* we need to write a z/os version of this, untill them return unknown */
```

Comment:
If this is for getprogrname, we added getprogname in zoslib here: https://github.com/ibmruntimes/zoslib/commit/42c179892b759b92cf20c6141d3271785ef3fe4a

Reply:
undone the patches/src/progname.c.patch